source/index-management/index-management.rst

```diff
@@ -888,7 +892,35 @@ An example, from Javascript:
   while (!(await collection.listSearchIndexes({ name }).hasNext())) {
     await setTimeout(1000);
   }
- 
+
+Where are read concern and write concern?
+-----------------------------------------
+
+These commands internally proxy the search index management commands to a separate process that runs alongside an Atlas cluster.  As such, read concern and 
+write concern are not relevant for the search index management commands.
+
+Consistency with Existing APIs
+------------------------------
+
+Drivers SHOULD strive for a search index management API that is as consistent with their existing search index management API as much as possible.
```

Comment:
This sentence is confusing - which APIs is it trying to differentiate?

Reply:
That was a typo - good catch